client/report_mirror

```diff
@@ -315,13 +305,10 @@ def main():
         sys.exit(1)
 
     if not options.no_send:
-        #   print "Connecting to %s" % item.config['global']['server']
+        #  print("Connecting to %s" % item.config['global']['server'])
         server = xmlrpclib.ServerProxy(item.config['global']['server'])
-        data = None
-        try:
-            data = base64.urlsafe_b64encode(bz2.compress(p))
-        except AttributeError:
-            data = urlsafe_b64encode(bz2.compress(p))
+
+        data = base64.urlsafe_b64encode(bz2.compress(p.encode())).decode()
 
```

Comment:
@pypingou this is the main change to make report_mirror python3 work with a server in python2. Does that make sense or is it just luck that it works with this change?

Reply:
Depending on the datatype it can be it :)